onboarding/src/screenshotTest/kotlin/io/homeassistant/companion/android/onboarding/connection/ConnectionErrorScreenshotTest.kt

```diff
@@ -0,0 +1,67 @@
+package io.homeassistant.companion.android.onboarding.connection
+
+import androidx.compose.runtime.Composable
+import com.android.tools.screenshot.PreviewTest
+import io.homeassistant.companion.android.common.R as commonR
+import io.homeassistant.companion.android.common.compose.theme.HAThemeForPreview
+import io.homeassistant.companion.android.compose.HAPreviews
+
+class ConnectionErrorScreenshotTest {
```

Comment:
Can/should we expand more details in one of the tests to make sure that also displays correctly?